tests/stdlib/tos.nim

```diff
@@ -261,6 +261,43 @@ block fileOperations:
 
     removeDir(dname)
 
+block: # moveFile
```

Comment:
don't write under cwd, that's bad style (eg this could overwrite existing files, or files from other tests; remember that testament runs in parallel)

instead, create a unique dir, eg:
```nim
block: # moveFile, moveDir
  let dir = getTempDir() / "D20210609T151608"
  createDir(dir)
  defer: removeDir(dir)
  # now only operate inside dir, eg:
  block: # moveFile
     writeFile(dir / "a.txt", "")
     ...
  block: # moveDir
     ...
```